nbextensions/bulkImportCell/tabs/configure.js

```diff
@@ -65,6 +77,61 @@ define([
             });
         }
 
+        /**
+         * Returns the output parameter values from the unselected file types,
+         * which will make it easier to look for duplicate values against the active file
+         * type.
+         *
+         * Since this is focused on being used for duplicate values, with a row to point
+         * to, it gets structured like this:
+         * {
+         *   value1: {
+         *     fileType: [rows],
+         *     fileType2: [rows]
+         *   },
+         *   value2: {
+         *     fileType: [rows]
+         *   }
+         * }
+         *
+         * The `fileType` fields above are the display name, just to make writing the alerts
+         * easier. This does come with the assumption that those names are all unique,
+         * but they really should be unless we want to be extra confusing to our users.
+         *
+         * Nulls, undefineds, and empty strings are ignored.
+         */
+        function getUnselectedOutputValues() {
+            const unselectedTypes = new Set(Object.keys(typesToFiles));
+            if (unselectedTypes.size === 1) {
+                return {}; // skip everything if there's only one file type
+            }
+
+            const unselectedOutputValues = {};
+            unselectedTypes.delete(selectedFileType);
+            const allParams = model.getItem('params');
+            const allOutputParamIds = model.getItem('app.outputParamIds');
+            unselectedTypes.forEach((type) => {
+                const outputParamIds = allOutputParamIds[type];
+                for (let i = 0; i < allParams[type].filePaths.length; i++) {
+                    outputParamIds.forEach((paramId) => {
+                        const value = allParams[type].filePaths[i][paramId];
+                        if (!value) {
+                            return;
+                        }
+                        if (!(value in unselectedOutputValues)) {
+                            unselectedOutputValues[value] = {};
+                        }
+                        const displayType = fileTypesDisplay[type].label;
+                        if (!(type in unselectedOutputValues[value])) {
```

Comment:
should that be `displayType`, rather than `type`?

Reply:
Yes! Oh, I wonder if that's part of the issue with rows not being right? That would overwrite this array each time it's made.